app/views/topics/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Topics</h2>
+<p>Add a new topic</p>
+
+<%= form_for @topic, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name, :required => 'required' %>
```

Comment:
Don't use browser validations. 
They are not comprehensive. 
They don't look pretty. 
They can be bypassed by a malicious user.